catalyst_voices/apps/voices/lib/widgets/text/timezone_date_time_text.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:catalyst_voices/widgets/common/affix_decorator.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+typedef DateTimeFormatBuilder = String Function(
+  BuildContext context,
+  DateTime dateTime,
+);
```

Comment:
A bit unfortunate wording, I couldn't understand from the name whether it should return formatted date or the format of the date.

Could we rename it so it's clearer what it should do?

```suggestion
typedef DateTimeFormatter = String Function(
  BuildContext context,
  DateTime dateTime,
);
```

Reply:
I thought about it but then we do have already [DateFormatter](https://github.com/input-output-hk/catalyst-voices/blob/main/catalyst_voices/apps/voices/lib/common/formatters/date_formatter.dart) which is even worse.

Maybe just `TimezoneDateTimeTextFormatter` ? 🥲

Reply:
Okay 👍